lib/madwf_transfer.cu

```diff
@@ -69,6 +69,8 @@ namespace quda
         errorQuda("ColorSpinorFields are not single parity: in = %d, out = %d", in.SiteSubset(), out.SiteSubset());
       }
 
+      if (out.Ndim() != 5 || in.Ndim() != 5) { errorQuda("we need a 5 dimensional field for this."); }
```

Comment:
I _think_ we also need to check and make sure `Ncolor == 3` and `Nspin == 4`

Reply:
I have added these checks in the `madwf_tensor/transfer.cu`, instead of in the corresponding `.cuh` files in https://github.com/lattice/quda/pull/1061/commits/88818f3007b5212873c44c3482a6bf2a5e67c9c3,